typescript/tests/test.ts

```diff
@@ -1,27 +1,27 @@
-import { expect } from "chai";
```

Comment:
Editorconfiguration is wrong. Prettier is not used. Nothing should be reformatted. Did you use the devcontainer?

Reply:
Fix the formatting merge conflict. I don't use the container. I use the default from whatever JetBrains uses. Is there any non-default formatting that the team committed to?

Reply:
If you install prettier, it takes care of everything on save.
For IntelliJ you can download the plugin here (not tested): https://www.jetbrains.com/help/idea/prettier.html